### PR TITLE
implicit conversions (views)

### PR DESCRIPTION
**Pull request by hibikir**

### CODE REVIEW

**Comment by ddgenome** [comment]
@slbahr what do you think of this one?

File: _posts/2015-07-29-implicit-conversions.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+ method.   
+
+So what does that {{ "implicit def" | sc: "implicitdef" }} mean?
+
+Any time we try to call a method that doesn't exist (or when a parameter doesn't match the expected type),
+ the compiler attempts to use a view to make it match.
+A view is a single-parameter function or constructor, declared with the {{ "implicit" | sc: "implicitdef" }} keyword in front of it. The  {{ "implicit" | sc: "implicitdef" }}  keyword tells the compiler
+that it can use this function automatically, for as long as it is in scope.
+
+It's almost as if Scala added methods without changing java.lang.String. No manual wrapping: it's almost invisible[1]. Sounds convenient!
```
Comment (ddgenome) [inline _posts/2015-07-29-implicit-conversions.md]: Is that `[1]` supposed to point somewhere?

**Comment by ddgenome** [comment]
If you crop the images to have slightly narrower widths, will the text size show up a little larger?

**Comment by ddgenome** [comment]
I dig this post!

I am going to submit a small PR to your repo @hibikir . Let's get this cleaned up a posted!

**Comment by ddgenome** [comment]
PR issued: https://github.com/hibikir/engineering-blog/pull/1

**Comment by ddgenome** [comment]
I added created another PR to your repo @hibikir 
https://github.com/hibikir/engineering-blog/pull/2

**Comment by slbahr** [comment]
Approved!

**Pull request merged**